db/migrate/20200112121946_fix_dns_timeout_setting.rb

```diff
@@ -0,0 +1,6 @@
+class FixDnsTimeoutSetting < ActiveRecord::Migration[5.2]
+  def up
+    Rails.cache.delete(Setting.cache_key("dns_timeout"))
+    Setting.find_by_name('dns_timeout')&.update_column(:value, nil) if Setting[:dns_timeout] == [nil]
```

Comment:
Since you're accessing the setting here, won't it be cached again?

Reply:
could be, i'll clear the cache again just in case.

Reply:
Feels like we need settings helpers in the migrations to do this cleanly. Not for this PR, but if we look at the db/migrate directory I'm sure we can come up with some helpers that generalize some things.